src/renderer/components/Editor/PlaybackManager.tsx

```diff
@@ -73,6 +79,11 @@ const PlaybackManager = ({
       return;
     }
 
+    if (rangeOverride !== null) {
```

Comment:
What does this logic do? Maybe add a comment for dumbos like me pls 😃

Reply:
Logic is, if we're playing from a deleted section instead of the main transcription (which is determined by rangeOverride !== null), then there should be no highlighted word so the nowPlayingWordIndex is set to -1

Reply:
would rather it be `null` instead of `-1` but theres quite a few places that assume it's strictly a number

Reply:
Commit that adds the comment: [a5fbed8](https://github.com/chloebrett/mlvet/pull/308/commits/a5fbed804fd6a3bda0362a781d0ae51f3272f17b)

And yeah, basically what Chloe said